src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+@import "../utils/variables";
+@import "../utils/mixins.scss";
+@import "../utils/reset.scss";
+
+.header {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: $whiteColor;
+  box-shadow: 8px 2px 4px rgba(0, 0, 0, 0.05);
+
+  .logo {
+    &__image {
+      display: block;
+      @include square(40px);
+    }
+  }
+
+  .nav {
+
+    &__list {
+      display: flex;
+      list-style: none;
+    }
+
+    &__item {
+      margin-right: 20px;
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
```

Comment:
you can also choose the right paddings for your link items to eliminate the problem with the tests